src/journal/JournalRecorder.cc

```diff
@@ -262,7 +279,7 @@ void JournalRecorder::create_next_object_recorder(
       new_object_recorder->get_object_number());
   }
 
-  new_object_recorder->append(append_buffers);
+  new_object_recorder->append(append_buffers, false);
```

Comment:
Not sure if I'm following. The splay offset lock is already acquired when this line is executed, and I cannot unlock it here, as it will be unlocked after this method returns